app/modules/import-export-customization/module.php

```diff
@@ -156,7 +156,7 @@ private function render_import_export_tab_content() {
 		if ( $is_cloud_kits_available ) {
 			$content_data['import']['button_secondary'] = [
 				'url' => Plugin::$instance->app->get_base_url() . '#/kit-library/cloud',
-				'text' => esc_html__( 'Open the Library', 'elementor' ),
+				'text' => esc_html__( 'Import from library', 'elementor' ),
```

Comment:
We made some changes afterwards but never realized it wasn't updated here as well since the old app's settings screen always loaded regardless of the experiment status (fixed in app/modules/import-export/module.php).